lib/cc/analyzer/container.rb

```diff
@@ -0,0 +1,98 @@
+require "posix/spawn"
+
+module CC
+  module Analyzer
+    class Container
+      DEFAULT_TIMEOUT = 15 * 60 # 15m
+
+      def initialize(
+        image:,
```

Comment:
@pbrisbin this is probably a ruby syntax I'm not familiar with, but just wanted to make sure the absence of a value here was intentional.